src/patients/care-goals/CareGoalForm.tsx

```diff
@@ -87,50 +85,68 @@ const CareGoalForm = (props: Props) => {
       </Row>
       <Row>
         <Column sm={12}>
-          <SelectWithLabelFormGroup
-            name="priority"
-            label={t('patient.careGoal.priority.label')}
-            isRequired
-            options={priorityOptions}
-            defaultSelected={priorityOptions.filter(({ value }) => value === priority)}
-            isEditable={!disabled}
-            isInvalid={!!careGoalError?.priority}
-            onChange={onPriorityChange}
-          />
+          <div className="form-group" data-testid="prioritySelect">
+            <Label
+              title="This is a required input"
+              text={t('patient.careGoal.priority.label')}
+              htmlFor={t('patient.careGoal.priority.label')}
```

Comment:
What is this `htmlFor` attribute doing? I checked out https://react-bootstrap.netlify.app/components/forms/#form-label-props, and it seems it should correspond to some form id. Maybe `prioritySelect`?

Reply:
Hey Drew, 
I believe `htmlFor` should match the `data-testid` property of the `form-group` div that encapsulates both the Label and Select component. See here: https://github.com/HospitalRun/hospitalrun-frontend/blob/master/src/shared/components/input/SelectWithLabelFormGroup.tsx. Cynthia and I can make that change!